src/components/Layout/DrawerNavigation.tsx

```diff
@@ -90,24 +87,15 @@ export const DrawerNavigationComponent: FunctionComponent<DrawerContentComponent
   navigation
 }) => {
   const { logout } = useLogout();
-  const { showAlert } = useContext(AlertModalContext);
+  const { showConfirmationAlert } = useContext(AlertModalContext);
   const showHelpModal = useContext(HelpModalContext);
   const { drawerButtons } = useDrawerContext();
   const handleLogout = useCallback((): void => {
     logout(navigation.dispatch);
   }, [logout, navigation.dispatch]);
 
   const onPressLogout = (): void => {
-    showAlert({
-      ...defaultConfirmationProps,
-      title: "Confirm logout?",
-      buttonTexts: {
-        primaryActionText: "Logout",
-        secondaryActionText: "Cancel"
-      },
-      visible: true,
-      onOk: handleLogout
-    });
+    showConfirmationAlert(new Error("confirmLogout"), handleLogout);
```

Comment:
Same for this~

Reply:
This has been refactored to:
```suggestion
    showConfirmationAlert(CONFIRMATION_MESSAGE.CONFIRM_LOGOUT, handleLogout)
```